public/components/application-ui-toggles/1-interactive.html

```diff
@@ -1,9 +1,17 @@
+<!--
+  @layer components {
+    .no-tap-highlihgt {
+      -webkit-tap-highlight-color: transparent;
+    }
+  }
+-->
+
 <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
 
 <label
   x-data="{ isChecked: false }"
   for="AcceptConditions"
-  class="relative h-8 w-14 cursor-pointer"
+  class="relative h-8 w-14 cursor-pointer [-webkit-tap-highlight-color:_transparent]"
```

Comment:
It doesn't matter too much but I'd rather see this as `[&::-webkit-tap-highlight-color:_transparent]`.

Reply:
I've tried this in [here](https://play.tailwindcss.com/VXZcs4P3WO) but the styles are not applied.